micrometer-core/src/main/java/io/micrometer/core/instrument/distribution/FixedBoundaryVictoriaMetricsHistogram.java

```diff
@@ -251,7 +251,7 @@ private void outputSummary(PrintStream printStream, double bucketScaling) {
         printStream.format("%14s %10s\n\n", "Bucket", "TotalCount");
 
         for (CountAtBucket bucket : nonZeroBuckets()) {
-            printStream.format(Locale.US, "%14.1f %10d\n", bucket.bucket() / bucketScaling, bucket.count());
+            printStream.format(Locale.US, "%14.1f %10f\n", bucket.bucket() / bucketScaling, bucket.count());
```

Comment:
Perhaps worth noting that this is an actual bug Error Prone caught, which I believe no one has reported because no one uses this outputSummary API on HistogramSnapshots. We should probably deprecate it unless I'm missing somewhere it is used.